.github/workflows/type_check.yml

```diff
@@ -0,0 +1,18 @@
+name: Steep Type Checking
+
+on: push
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Nit: You can use a [newer version of checkout](https://github.com/actions/checkout)

Reply:
Ah, I'll ticket that and handle it immediately afterward, thanks :-)